test/facade_converter_test.ts

```diff
@@ -94,32 +94,46 @@ describe('type based translation', () => {
     it('translates array operations to dartisms', () => {
       expectWithTypes('var x: Array<number> = []; x.push(1); x.pop();')
           .to.equal(' List < num > x = [ ] ; x . add ( 1 ) ; x . removeLast ( ) ;');
+
```

Comment:
Why the newlines here, but not below? I think I'd rather not have many newlines, but group related blocks of tests within this `it` by a newline.